types/echarts/index.d.ts

```diff
@@ -28,6 +26,15 @@ declare namespace echarts {
     function registerMap(mapName: string, geoJson: Object, specialAreas?: Object): void;
 
     function registerTheme(themeName: string, theme: Object): void;
+    
+    interface mapObj {
+        // 地图的 geoJson 数据
```

Comment:
Please consider using JSDoc style comment for field comments. Also, English should be preferred.